litestar/contrib/repository/__init__.py

```diff
@@ -1,14 +1,20 @@
-from __future__ import annotations
-
-from .abc import AbstractAsyncRepository, AbstractSyncRepository
-from .exceptions import ConflictError, NotFoundError, RepositoryError
-from .filters import FilterTypes
-
-__all__ = (
-    "AbstractAsyncRepository",
-    "AbstractSyncRepository",
-    "ConflictError",
-    "FilterTypes",
-    "NotFoundError",
-    "RepositoryError",
-)
+from litestar.utils import warn_deprecation
+
+
+def __getattr__(attr_name: str) -> object:
+    from litestar import repository
+
+    for k in repository.__all__:
+        warn_deprecation(
+            deprecated_name=f"litestar.contrib.repository.{k}",
+            version="2.1",
+            kind="import",
+            removal_in="3.0",
+            info=f"importing {attr_name} from 'litestar.contrib.repository' is deprecated, please"
```

Comment:
```suggestion
            alternative=f"litestar.repository.{k}"
```

applies to the other files as well.

Reply:
done